packages/pyright-internal/src/analyzer/program.ts

```diff
@@ -391,23 +382,22 @@ export class Program {
         return sourceFile;
     }
 
-    setFileOpened(filePath: string, version: number | null, contents: string, options?: OpenFileOptions) {
-        let sourceFileInfo = this.getSourceFileInfo(filePath);
+    setFileOpened(fileUri: Uri, version: number | null, contents: string, options?: OpenFileOptions) {
+        let sourceFileInfo = this.getSourceFileInfo(fileUri);
         if (!sourceFileInfo) {
-            const moduleImportInfo = this._getModuleImportInfoForFile(filePath);
+            const moduleImportInfo = this._getModuleImportInfoForFile(fileUri);
             const sourceFile = this._sourceFileFactory.createSourceFile(
                 this.serviceProvider,
-                filePath,
+                fileUri,
                 moduleImportInfo.moduleName,
                 /* isThirdPartyImport */ false,
                 moduleImportInfo.isThirdPartyPyTypedPresent,
                 this._editModeTracker,
                 this._console,
                 this._logTracker,
-                options?.realFilePath,
                 options?.ipythonMode ?? IPythonMode.None
             );
-            const chainedFilePath = options?.chainedFilePath;
+            const chainedFilePath = options?.chainedUri;
```

Comment:
Should `chainedUri` be `chainedFileUri`?

Reply:
Sure, I don't think it matters much. I guess chainedFileUri indicates it's a file though. I'll change it.